### PR TITLE
Fix responsive issues in full header

### PR DESCRIPTION
**Pull request by anshumanv**
- SideMenu at 768
- Handle breaking of the header

### CODE REVIEW

File: src/components/AppHeader/UserMenu.js [hunk 1]
```diff
@@ -63,7 +69,7 @@ const UserMenu = ({ user }) => {
 					{user.name && user.name.charAt(0).toLocaleUpperCase()}
 				</Avatar>
 				&nbsp;&nbsp;
-				{user ? user.name : 'Loading...'}
+				<span className="userName">{user ? user.name : 'Loading...'}</span>
```
Comment (jyash97) [inline src/components/AppHeader/UserMenu.js]: `css` prop returns a unique className based on the styles with the help of `@emotion/hash`.
```suggestion
				<span className={userName}>{user ? user.name : 'Loading...'}</span>
```

**Review by jyash97 — CHANGES_REQUESTED** [review]
Looks good, except for a small change in `className`.

File: src/components/AppHeader/UserMenu.js (hunk 1 above)
Comment (anshumanv) [reply src/components/AppHeader/UserMenu.js]: Missed that one, thanks! :)

**Comment by anshumanv** [comment]
Here are some samples - 

image

image

image

image

**Pull request merged**